tutorials/pushing-metrics-logs-from-scw-instance/index.mdx

```diff
@@ -0,0 +1,134 @@
+---
+meta:
+  title: How to push metrics and logs from a Scaleway Instance
+  description: This page shows you how to push metrics and logs from a Scaleway Instance
+content:
+  h1: How to push metrics and logs from a Scaleway Instance
+  paragraph: This page shows you how to push metrics and logs from a Scaleway Instance
+tags: cockpit observability metrics logs instance
+categories:
+  - observability
+  - compute
+  - cockpit
+  - instances
+dates:
+  validation: 2023-06-20
+  posted: 2023-06-20
+---
+
+This page shows you how to push your metrics and logs from a [Scaleway Instance](/compute/instances/concepts/#instance).
+
+<Macro id="iam-requirements" />
+ 
+<Message type="requirement">
+ - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+ - You have activated your [Cockpit](/observability/cockpit/how-to/activate-cockpit)
+</Message>
+
+<Message type="important">
+ - Having the default configuration on your agents might lead to more of your resources’ metrics being sent, a high consumption and a high bill at the end of the month.
+ - Sending metrics and logs for Scaleway resources or personal data using an external path is a billable feature. In addition, any data that you push yourself is billed, even if you send data from Scaleway products. Refer to the [product pricing](https://www.scaleway.com/en/pricing/?tags=available,managedservices-observability-cockpit) for more information.
+</Message>
+
+## Configure your environment
+
+1. [Create an Instance](/compute/instances/how-to/create-an-instance) from the **Compute** section of the Scaleway console.
+2. Copy your Instance's public IP as you will need it in the next steps.
+3. Open a terminal and type the following command to connect to your Instance:
+
+            ```bash
+            ssh root@<your Instance public IP>
+            ```
+4. Open another terminal and type the following command to clone our workshop repository on your local machine:
+
+            ```bash
+            git clone git@gitlab.infra.online.net:observability-paas/workshop.git
```

Comment:
Client will not have access to this Gitlab.
You can move the project in the public github.

Reply:
Should we create a specific public github to use for these kinds of tutos that involve Obs?

Reply:
I created a repository "tuto-cockpit-instance" in which I put the files